code/game/machinery/airlock_cycle_control.dm

```diff
@@ -778,7 +793,7 @@
 			locked = !locked
 			update_icon()
 			to_chat(user, "<span class='notice'>You [ locked ? "lock" : "unlock"] the airlock controller interface.</span>")
-			updateUsrDialog()
+			ui_update()
```

Comment:
This call is kind of redundant since the advanced airlock controller auto updates anyway